test/testdata/lsp/code_actions/move_method/comments.rb

```diff
@@ -0,0 +1,29 @@
+# typed: strict
+# selective-apply-code-action: refactor.extract
+#
+# Test asserts comments between a sig and a method def are not moved,
+# because we cut out only a signature and a method body without touching anything in between
```

Comment:
Are we sure that this is the desired behavior? It seems to me that it isn't desired for a number of reasons:

- People will lose potentially important comments, and have to move them on their own
- Modifiers like `private_class_method` that exist between the `sig` and the `def` will not be moved.

It seems that if we change the code to call `sigLoc.join(methodDefLoc)` to compute the source of the method to be moved, we solve these issues and **also** solve the issue that we're leaving so many empty lines behind at the old position in the file (it won't get rid of all of the empty lines, but there will at least be fewer, which is a start).

LMK if you made this choice as a conscious design decision, because I'm curious to hear the justification.

Reply:
That was a design choice, but more I work on that PR, more I think we should leave it. The main consideration was, I wanted to be as careful as possible, and make the least amount of mess with the refactoring. 

Moving everything between the sig and the method would probably make more sense, as this behavior is probably more obvious for the user.